### ISSUE TITLE
CAS 5.0 Hazelcast deprecated properties

### ISSUE THREAD

**Issue by tommendenhall** [apereo/cas#2052]
CAS 5.0 RC4 snapshot
Centos 7, openJdk 1.8.0, Tomcat 8.5.6

During tomcat start.
catalina.out

```bash
2016-10-17 08:08:49,942 WARN [com.hazelcast.config.MapConfig] - <As of version 3.7, `minEvictionCheckMillis` and `evictionPercentage` are deprecated 
due to the eviction mechanism change. New eviction mechanism uses a probabilistic algorithm based on sampling. Please see documentation for further details>
```

cas.properties

```properties
# hazelcast ticket cache replication
cas.ticket.registry.hazelcast.pageSize=500
cas.ticket.registry.hazelcast.mapName=tickets
cas.ticket.registry.hazelcast.cluster.evictionPolicy=LRU
cas.ticket.registry.hazelcast.cluster.maxNoHeartbeatSeconds=300
cas.ticket.registry.hazelcast.cluster.multicastEnabled=false
cas.ticket.registry.hazelcast.cluster.evictionPercentage=10
cas.ticket.registry.hazelcast.cluster.tcpipEnabled=true
cas.ticket.registry.hazelcast.cluster.members=192.168.118.7,192.168.118.8,192.168.118.9
cas.ticket.registry.hazelcast.cluster.loggingType=slf4j
cas.ticket.registry.hazelcast.cluster.instanceName=localhost
cas.ticket.registry.hazelcast.cluster.port=5701
cas.ticket.registry.hazelcast.cluster.portAutoIncrement=true
cas.ticket.registry.hazelcast.cluster.maxHeapSizePercentage=85
cas.ticket.registry.hazelcast.cluster.backupCount=1
cas.ticket.registry.hazelcast.cluster.asyncBackupCount=0
cas.ticket.registry.hazelcast.cluster.maxSizePolicy=USED_HEAP_PERCENTAGE
cas.ticket.registry.hazelcast.cluster.timeout=5
# cas.ticket.registry.hazelcast.crypto.signing.key=
# cas.ticket.registry.hazelcast.crypto.signing.keySize=512
# cas.ticket.registry.hazelcast.crypto.encryption.key=
# cas.ticket.registry.hazelcast.crypto.encryption.keySize=16
# cas.ticket.registry.hazelcast.crypto.alg=AES
```

log4j2.xml

```xml
       <AsyncLogger name="com.hazelcast" level="info" additivity="false">
            <AppenderRef ref="casConsole"/>
            <AppenderRef ref="casFile"/>
```

**Comment by mmoayyed** [comment]
First, thanks very much. 

Second, please use backticks such as ``` to format settings, etc in code blocks. Makes it easier for everyone to read. (I formatted yours, see that for an example) If you fail to do so, you report will show up half complete cause a bunch of stuff gets "hidden away".

**Comment by mmoayyed** [comment]
Solved in the new 5.1 SNAPSHOT
